dbt_subprojects/hourly_spellbook/models/_project/safe/scroll/safe_scroll_schema.yml

```diff
@@ -0,0 +1,167 @@
+version: 2
+
+sources:
+  - name: gnosis_safe_scroll
+    tables:
+      - name: GnosisSafeProxyFactory_v1_3_0_evt_ProxyCreation
+      - name: SafeProxyFactory_v1_4_1_evt_ProxyCreation
```

Comment:
sources should live [here](https://github.com/duneanalytics/spellbook/tree/main/sources/safe), initiate new directory/file for scroll